src/index.ts

```diff
@@ -276,6 +276,21 @@ async function launchMetals(
       });
     });
 
+    registerCommand("metals.analyze-stacktrace", async () => {
+      const trace: string = await workspace.nvim.call("getreg", "*");
+      if (trace.trim().length < 1) {
+        workspace.showMessage(
+          "No text found in your register. Copy your stacktrace to your register and retry.",
+          "warning"
+        );
+      } else {
+        client.sendRequest(ExecuteCommandRequest.type, {
+          command: "analyze-stacktrace",
```

Comment:
I think scalameta/metals-languageclient#184 should actually be used here as it points to ServerCommand

Reply:
- [ ] Add this in from metals-languageclient after release.